data/core/tips.json

```diff
@@ -272,28 +262,18 @@
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Thirsty on a long winter journey?  Take a thermos with you." ]
+    "text": [ "A thermos can keep its contents at a constant temperature.  Useful to stop water freezing in winter." ]
```

Comment:
Again, this is completely valid still, and the original delivery was less dry.

Reply:
The old one didn't actually explain the mechanic at all.

Reply:
The fact that a thermos keeps a liquid a liquid doesn't really have to be explained IMHO.